webapp/src/ts/app.component.html

```diff
@@ -14,12 +14,9 @@
 
   <mm-actionbar *ngIf="useOldActionBar" [reportForms]="reportForms"></mm-actionbar>
 
-  <div class="container-fluid" *ngIf="initialisationComplete">
-    <mm-header
-      [adminUrl]="adminUrl"
-      [canLogOut]="canLogOut"
-    ></mm-header>
-    
+  <div class="container-fluid" *ngIf="initialisationComplete" [class.old-nav]="!useOldNav">
```

Comment:
The `old-nav` class is applied when the auth.has(`can_view_old_navigation`) is true.

```suggestion
  <div class="container-fluid" *ngIf="initialisationComplete" [class.old-nav]="useOldNav">
```